server/db/seeds.rb

```diff
@@ -3,16 +3,16 @@
 Programminglanguage.destroy_all
 Wish.destroy_all
 users = ApiUser.create([
-                {first_name: 'Mary', last_name: 'Black', city: 'Dublin', mentor: true, mentee: false, email: 'Test_email_1@example.com', password_digest: 'password1'}, 
-              {first_name: 'Bob', last_name: 'Dylan', city: 'New York', mentor: true, mentee: true, email: 'Test_email_2@example.com', password_digest: 'password2'}, 
-               {first_name: 'Jane', last_name: 'Smith', city: 'Edinburgh', mentor: false, mentee: true, email: 'Test_email3@example.com', password_digest: 'password3'}
+                {first_name: 'Mary', last_name: 'Black', city: 'Dublin', mentor: true, mentee: false, email: 'Test_email_1@example.com', password_digest: 'password1', username: 'user1'}, 
+              {first_name: 'Bob', last_name: 'Dylan', city: 'New York', mentor: true, mentee: true, email: 'Test_email_2@example.com', password_digest: 'password2', username: 'user2'}, 
+               {first_name: 'Jane', last_name: 'Smith', city: 'Edinburgh', mentor: false, mentee: true, email: 'Test_email3@example.com', password_digest: 'password3', username: 'user3' }
                ])
 Meetinginterval.create([{interval: 'weekly'}, {interval: 'daily'}])
 Programminglanguage.create([{language: 'python'},{language: 'SCALA'}])
 
-users.first.wishes.create([{available_offline: false, available_online: true, goal: 'Code real good'}])
-users.second.wishes.create([{available_offline: true, available_online: true, goal: 'Get smarter'}])
-users.first.wishes.create([{available_offline: false, available_online: true, goal: 'Code really well in Python'}])
+users.first.wishes.create([{available_offline: false, available_online: true, goal: 'Code real good', programminglanguage_id: Programminglanguage.second.id, meetinginterval_id: Meetinginterval.first.id}])
```

Comment:
No need to explicit have the IDs:

`programminglanguage: Programminglanguage.first, meetinginterval: Meetinginterval.first`

Reply:
Also, since there is no order on the `Meetinginterval` model, using `Meetinginterval.first` and `Meetinginterval.second` might yield different results depending on the database/version etc. 

A more robust way would be to define them one by one:
```
daily_meetinginterval = Meetinginterval.create({interval: 'daily'})
weekly_meetinginterval = Meetinginterval.create({interval: 'weekly'})
```

and then you can do it like:
```
users.first.wishes.create([{available_offline: false, available_online: true, goal: 'Code real good', programminglanguage: Programminglanguage.second, meetinginterval: weekly_meetinginterval}])
```

Same problem is for programming language.

Reply:
Changed

Reply:
This change is included in pull request #216